crates/client/telemetry/src/lib.rs

```diff
@@ -18,42 +14,33 @@ pub enum VerbosityLevel {
     Debug = 1,
 }
 
-#[derive(Debug)]
+#[derive(Debug, Clone)]
 pub struct TelemetryEvent {
     verbosity: VerbosityLevel,
     message: serde_json::Value,
 }
 
 #[derive(Debug, Clone)]
-pub struct TelemetryHandle(Option<Arc<mpsc::Sender<TelemetryEvent>>>);
+#[repr(transparent)]
+pub struct TelemetryHandle(tokio::sync::broadcast::Sender<TelemetryEvent>);
```

Comment:
ah! why the change?

Reply:
So! We need services to be able to be potentially restarted. This does not work if we store `TelemetryHandle` as an `Option` and just call `std::mem::take` on it. Similarly, we cannot use a `tokio::sync::mpsc` as we need to be able to re-create the receiver on each service start (the service future takes ownership of it).